transport/spi/pom.xml

```diff
@@ -72,6 +72,11 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>compile</scope>
```

Comment:
I didn't actually add this element, I just moved it to be before jmock-junit4 because otherwise you can't run  junit test methods from Eclipse. It already had compile scope. That is needed because there is testware under src/main that uses junit classes (like TransportTestIoHandlerAdapter)